doc/rados/operations/monitoring.rst

```diff
@@ -626,6 +626,21 @@ For example, the following commands are equivalent to each other:
    ceph daemon osd.0 foo
    ceph daemon /var/run/ceph/ceph-osd.0.asok foo
 
+There are two methods of running admin socket commands: (1)
+using ``ceph daemon`` as described above, which bypasses
+the monitor and assumes a direct login to the daemon's host,
+and (2) using the ``ceph tell {daemon-type}.{id}`` command,
+which is relayed by the monitor and does not require access
+to the daemon's host.
```

Comment:
Suggestion:  add an containerized example where the `.asok` file is under `/var/lib/something`

Reply:
There is an example at line 622
```
   ceph daemon /var/run/ceph/ceph-osd.0.asok foo
```
Isn't that sufficient?

Reply:
Someone running containerized will find nothing under /var/run/ceph

Reply:
My plan was to only mention the new command while leaving the rest without modifications. If you'd like to use this as an opportunity to improve the adjacent text, be my guest, but I'd have to ask you to suggest the actual edits. Thanks!